include/meson.build

```diff
@@ -9,7 +9,9 @@ headers = [
   'downloader.h',
   'reader.h',
   'entry.h',
-  'searcher.h'
+  'searcher.h',
+  'kiwixserve.h',
+  'subprocess.h'
```

Comment:
This is the list of header files to install.
Header we don't want to install (not part of the public API) don't need (and must not) to be installed.
So we don't need `subprocess.h` here.